code/src/components/subintro/Subintro.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import Subintro from "@/components/subintro/Subintro";
+import  subintroData  from "@/components/subintro/Subintro.constants";
+jest.mock("@/assets/images/subintro/moped-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/box-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/percent-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/clock-icon.jpg", () => { });
+
+describe("Subintro Component", () => {
+  test("render number length of subintro elements", () => {
```

Comment:
I suggest refactoring to `render all subintro elements correctly`

Reply:
Done